lib/ansible/modules/find.py

```diff
@@ -515,7 +515,7 @@ def main():
     skipped = {}
 
     def handle_walk_errors(e):
-        if e.errno in (errno.EPERM, errno.EACCES):
+        if e.errno in (errno.EPERM, errno.EACCES, errno.ENOENT):
```

Comment:
should we warn about ENOENT? since user might not know files are disappearing from under them

Reply:
we show a warning on top - 
```
[WARNING]: Skipped entry '/proc/190/task/190/fd/3' due to this access issue: [Errno 2]
No such file or directory: '/proc/190/task/190/fd/3'
```
also 

```
"skipped_paths": {"/proc/190/fd/3": "[Errno 2] No such file or directory: '/proc/190/fd/3'",
```